src/psyclone/transformations.py

```diff
@@ -250,11 +273,11 @@ def apply(self, node1, node2):
         keep = Memento(schedule, self, [node1, node2])
 
         # add loop contents of node2 to node1
-        node1.children.extend(node2.children)
+        node1.loop_body._children.extend(node2.loop_body)
```

Comment:
pylint's not going to like the use of `_children`. Is there a `children` property we can use instead?

Reply:
Sure